classes/core/PKPRequest.inc.php

```diff
@@ -591,6 +591,18 @@ function &getUser() {
 	function getUserVar($key) {
 		$_this = PKPRequest::_checkThis();
 
+		if (is_a($this, 'Request')) {
```

Comment:
OK. I think `$this` might be used elsewhere; when using `$_this`, I don't think a check is necessary (and would rather a hard error if `$_this` isn't set).